episodes/chapter4.md

```diff
@@ -0,0 +1,242 @@
+
+4. Click on the `Submit` button to create the new collaboration.
+    * After submitting the form, you will see a dialog box to ask you to download the API key. The API key is used to authenticate the nodes in the collaboration. Later we will see how to reset the API key for a node.
+
+![Download API keys](fig/create_collab_02.png)
+
+You will see a message:
+
+> API key download
+>
+> The API keys have been downloaded.
+>
+> Please distribute each of these keys privately to each of the organizations. Note that they may reset their API key so that no-one but them knows it.
+
+5. After creating the collaboration, you can see the details of the collaboration by clicking on the specific collaboration listed in the `Collaborations` tab.
+    * You will see what organizations are participating in the collaboration.
+    * You will see the nodes of the organizations that are registered in the collaboration. If no nodes are registered, you can register them manually by clicking the `Register missing nodes` button.
+    * You can also see the algorithm store available for the collaboration. You can add a algorithm store for the collaboration by clicking the `Add algorithm store` button.
+    * You can also see the studies of the collaboration. You can add a study by clicking the `Add study` button.
+
+![Collaboration details](fig/create_collab_03.png)
+
+### Reset API key for a node
+If you want to reset the API key for a node, you can do so by following these steps:
+
+1. Click on the `Nodes` tab in the administration page.
+2. Click on the tab of the node you want to reset the API key for in the list of nodes.
+
+![Reset API key for a node](fig/reset_api_key_01.png)
+
+3. Click on the `Reset API key` button.
+    * You will see a dialog box to ask you to download the new API key.
+
+![Download new API key](fig/reset_api_key_02.png)
+
+You will see a message:
+
+> API key download
+>
+> Your API key has been reset. Please read your new key in the file that has been downloaded.
+
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 3: Manage Vantage6 project using the UI
+
+Go the administration page in the Vantage6 UI and try to create a new organization, a new user, and a new collaboration. Also, try to reset the API key for a node.
+
+1. Can you create a new organization, a new user, and a new collaboration?
+2. Can you reset the API key for a node?
+3. Which steps are challenging for you?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+::::::::::::::::::::::::::::::::::::: keypoints
+
+- Understand Vantage6 entities including `Users`, `Organizations`, `Collaborations`, `Roles`, `Nodes`, and `Tasks`
+- Understand Vanatge6 permission system and default roles such as `Root`, `Collaboration Admin`, `Organization Admin`, `Researcher`, and `Viewer`
+- Be able to create a new organization, a new user, and a new collaboration using the Vantage6 UI
+- Be able to reset an api key for a node
+
```

Comment:
These key points are basically the same as the objectives. For key points, aren't we supposed to say stuff like 'The main vantage6 entities are ...'?